### ISSUE TITLE
Segregate code into api files

### ISSUE THREAD

**Issue by kevinrodriguezGL** [gorillalogic/winston-lambda#6]
We should create a folder named `/lib/api`
and add all of the api js files there.

We should also create a new called helpers under lib `/lib/helpers`.

**Comment by greivinlopezgl** [comment]
I agree but let's remove the "lib" folder from the equation to make it simpler.  Let's discuss the refactor tomorrow.

**Comment by greivinlopezgl** [comment]
I agree let's have a call to discuss this redistribution of source code.
